src/Network/Wai/SAML2/Request.hs

```diff
@@ -33,6 +33,7 @@ import Crypto.Random
 
 import Data.Time.Clock
 import Data.Time.Format
```

Comment:
Looks like this import might be redundant now.

Reply:
good catch